### ISSUE TITLE
Demo app UI not refreshing on gulp build:watch

### ISSUE THREAD

**Issue by edetec** [tinesoft/generator-ngx-library#149]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version? 
Fedora 27 x86_64

### Generator Version
ngx-library@5.5.0 

### Tools Version
yo 2.0.0
node v8.9.4
npm 5.6.0

### Targeted Angular version

Angular 5.2.0
### Repro steps.

1. running gulp build:watch-fast and gulp serve:demo simultaneously

2. change lib UI

### The log given by the failure.
The demo is rebuilt but do not have last changes of lib

### Desired functionality.
The demo should have the last changes of lib

### Mention any other details that might be useful.
https://github.com/ESSS/ng-xform

**Comment by AnthonyNahas** [comment]
can you try please instead of `gulp:serve:demo` `ng serve` in the demo directory ?
workflow:
In the first terminal run `gulp build:watch_fast`
In the second terminal run `cd demo && ng serve`

?

**Comment by tinesoft** [comment]
@edetec:  You mean you removed the `--aot` option? from `gulp serve:demo` task to have the problem solved?

Can you share the **link to your discoveries** on angular problems related to the `--aot` option?

Thanks in advance

**Comment by edetec** [comment]
of course, I think is that https://github.com/angular/angular-cli/issues/6347

**Comment by tinesoft** [comment]
ok thanks!

i'll keep a eye on it
